packages/fuels-contract/src/contract.rs

```diff
@@ -438,6 +443,12 @@ impl ContractCall {
         let decoded_value = ABIDecoder::decode_single(&self.output_param, &encoded_value)?;
         Ok(decoded_value)
     }
+
+    fn check_if_contract_not_in_inputs(receipts: &[Receipt]) -> Option<&Receipt> {
```

Comment:
Nit: for me `check_` usually implies that the method is returning a boolean. Maybe something like `find_contract_not_in_inputs` is more descriptive.